External/bids-matlab/+bids/layout.m

```diff
@@ -380,6 +380,9 @@ function tolerant_message(tolerant, msg)
                             m0_filebase = bids.internal.file_utils(bids.internal.file_utils(subject.perf(j).filename, 'basename'), 'basename'); % remove the extension
                             m0_filebase = m0_filebase(1:end-4); % to remove the _asl suffix
                             m0_filename = bids.internal.file_utils('List', pth, sprintf('^%s.*_m0scan\\.nii(\\.gz)?$', m0_filebase)); % find the file
+							if isempty(m0_filename)
```

Comment:
As discussed in person - there was a serious bug that prevented from testing this entire branch. I would prefer to keep it here. If you really really want it separate. I can remove this before final merging and create a new issue - but that's a lot of unnecessary work and you can see the changes actually easily by comparing 1.8.0 with develop and looking at the BIDS folder.

Decision?

Reply:
@HenkMutsaerts totally understand what you mean. The thing is that we couldn't test a lot of the datasets with the previous code. @jan-petr did a really nice fix there. Instead of modifying bids-matlab any further, we agreed to just update the whole thing in a future/next release. IMHO this should be the correct way to do this.